diagnostics-extension/src/telemetry.service.test.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
```

Comment:
we named other test files with one of the following spec or unit or it, prefer consistency.

Reply:
The reason for naming them so is: The testing library for extension, `Jest` prefers `.test.ts` convention. While the Karma + Jasmine in Angular prefers `.spec.ts`. But since they're in the same repository, they should be consistent. :+1: